src/NZGBplugin/Layers.py

```diff
@@ -609,9 +621,9 @@ def deleteSelectedGeometries(self):
             return
 
         gtypes = (
-            (QGis.Point, "point", "fpoint"),
-            (QGis.Line, "line", "fline"),
-            (QGis.Polygon, "polygon", "fpoly"),
+            (QgsWkbTypes.Point, "point", "fpoint"),
```

Comment:
Is this some other QGIS 2->3 API changes?

Reply:
All methods taking or returning QGis::WkbType have been changed to use QgsWkbTypes::Type 
(https://qgis.org/api/api_break.html)